### DISCUSSION TITLE
Monitor won't turn off while SMPlayer is running

### DISCUSSION THREAD

**Discussion by yolanjoy** [smplayer-dev/smplayer#1040] [General]
https://old-forum.smplayer.info/viewtopic.php?f=2&t=113&sid=9aca2cc3479c8c23cf01aa75b48eebe5&start=30

Same issue described in the old forum.  Not sure if this was fixed or any workaround out there.

Thanks

**Comment by Raza-0007** [comment]
Isn't this by design? Surely you don't want the screen to turn off while you are watching a movie? 

If you are playing an audio file and want the screen to turn off, then uncheck the Screensaver options under "Preferences -> General -> Video" and make sure smplayer is in the foreground.

~Raza

**Comment by yolanjoy** [comment]
Monitor won't turn off even when SMPlayer isn't running any video/audio.

**Reply by Raza-0007** [reply]
That is strange. It does not happen with me. My screen turns off when smplayer is open but not playing anything. Try with a portable version of smplayer, or with your current settings reset to default and see if your issue persists. 

~Raza

**Comment by yolanjoy** [comment]
With the portal version, the monitor does turn off while SMPlayer is running.

**Comment by attila-lendvai** [comment]
my issue is that smplayer inhibits the screensaver even when the video is paused (on linux).

as discussed here: http://old-forum.smplayer.info/viewtopic.php?f=2&t=7172&sid=858796346a334ff896dfed317d87ff79&start=10

there's a way to change mvp config like this:

```
nostop-xscreensaver=1
heartbeat-interval=60
heartbeat-cmd="xdotool key VoidSymbol"
```

but i guess mpv is not reading its `~/.config/mpv/mpv.conf` file when it is smplayer's backend, because this doesn't work (i.e. if i uncheck the disable screensave in the smplayer config, then the screensaver activates even while the video is playing).

there's an issue and a PR for this:
https://github.com/smplayer-dev/smplayer/pull/641
https://github.com/smplayer-dev/smplayer/issues/582
